extra semicolons

I'm on it
```
goodrich@T540p:/opt/stan$ grep -F "warning:" /tmp/travis.txt | sort | uniq | sed 's@/usr/local/lib/R/site-library/StanHeaders/include@src@g'
src/stan/lang/grammars/functions_grammar_def.hpp:40:2244: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/statement_grammar_def.hpp:48:1773: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/statement_grammar_def.hpp:52:1767: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/statement_grammar_def.hpp:56:2312: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/statement_grammar_def.hpp:61:1273: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/statement_grammar_def.hpp:64:1273: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/statement_grammar_def.hpp:67:1281: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/statement_grammar_def.hpp:70:1361: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/statement_grammar_def.hpp:73:2234: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/statement_grammar_def.hpp:78:1788: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/term_grammar_def.hpp:53:4468: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/term_grammar_def.hpp:66:1729: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/term_grammar_def.hpp:70:1762: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/bare_type_grammar_def.hpp:28:1737: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/functions_grammar_def.hpp:34:2925: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/statement_grammar_def.hpp:44:1752: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/term_grammar_def.hpp:48:1750: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]
src/stan/lang/grammars/term_grammar_def.hpp:62:1673: warning: extra ';' outside of a function is a C++11 extension [-Wc++11-extra-semi]

```

--- a/src/stan/lang/grammars/bare_type_grammar_def.hpp
+++ b/src/stan/lang/grammars/bare_type_grammar_def.hpp
@@ -27,7 +27,7 @@
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::expr_type,
                           (stan::lang::base_expr_type, base_type_)
-                          (size_t, num_dims_) );
+                          (size_t, num_dims_) )
 
 namespace stan {
 

--- a/src/stan/lang/grammars/functions_grammar_def.hpp
+++ b/src/stan/lang/grammars/functions_grammar_def.hpp
@@ -35,12 +35,12 @@ BOOST_FUSION_ADAPT_STRUCT(stan::lang::function_decl_def,
                           (stan::lang::expr_type, return_type_)
                           (std::string, name_) 
                           (std::vector<stan::lang::arg_decl>, arg_decls_)
-                          (stan::lang::statement, body_) );
+                          (stan::lang::statement, body_) )
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::arg_decl,
                           (stan::lang::expr_type, arg_type_)
                           (std::string, name_)
-                          (stan::lang::statement, body_) );
+                          (stan::lang::statement, body_) )
 
 namespace stan {
 

--- a/src/stan/lang/grammars/statement_grammar_def.hpp
+++ b/src/stan/lang/grammars/statement_grammar_def.hpp
@@ -43,41 +43,41 @@
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::assignment,
                           (stan::lang::variable_dims, var_dims_)
-                          (stan::lang::expression, expr_) );
+                          (stan::lang::expression, expr_) )
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::variable_dims,
                           (std::string, name_)
-                          (std::vector<stan::lang::expression>, dims_) );
+                          (std::vector<stan::lang::expression>, dims_) )
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::distribution,
                           (std::string, family_)
-                          (std::vector<stan::lang::expression>, args_) );
+                          (std::vector<stan::lang::expression>, args_) )
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::for_statement,
                           (std::string, variable_)
                           (stan::lang::range, range_)
-                          (stan::lang::statement, statement_) );
+                          (stan::lang::statement, statement_) )
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::return_statement,
-                          (stan::lang::expression, return_value_) );
+                          (stan::lang::expression, return_value_) )
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::print_statement,
-                          (std::vector<stan::lang::printable>, printables_) );
+                          (std::vector<stan::lang::printable>, printables_) )
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::reject_statement,
-                          (std::vector<stan::lang::printable>, printables_) );
+                          (std::vector<stan::lang::printable>, printables_) )
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::increment_log_prob_statement,
-                          (stan::lang::expression, log_prob_) );
+                          (stan::lang::expression, log_prob_) )
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::sample,
                           (stan::lang::expression, expr_)
                           (stan::lang::distribution, dist_) 
-                          (stan::lang::range, truncation_) );
+                          (stan::lang::range, truncation_) )
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::statements,
                           (std::vector<stan::lang::var_decl>, local_decl_)
-                          (std::vector<stan::lang::statement>, statements_) );
+                          (std::vector<stan::lang::statement>, statements_) )
 
 namespace stan {
 

--- a/src/stan/lang/grammars/term_grammar_def.hpp
+++ b/src/stan/lang/grammars/term_grammar_def.hpp
@@ -48,7 +48,7 @@
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::index_op,
                           (stan::lang::expression, expr_)
                           (std::vector<std::vector<stan::lang::expression> >, 
-                           dimss_) );
+                           dimss_) )
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::integrate_ode,
                           (std::string, system_function_name_)
@@ -57,19 +57,19 @@ BOOST_FUSION_ADAPT_STRUCT(stan::lang::integrate_ode,
                           (stan::lang::expression, ts_)
                           (stan::lang::expression, theta_)
                           (stan::lang::expression, x_)
-                          (stan::lang::expression, x_int_) );
+                          (stan::lang::expression, x_int_) )
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::fun,
                           (std::string, name_)
-                          (std::vector<stan::lang::expression>, args_) );
+                          (std::vector<stan::lang::expression>, args_) )
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::int_literal,
                           (int,val_)
-                          (stan::lang::expr_type,type_));
+                          (stan::lang::expr_type,type_))
 
 BOOST_FUSION_ADAPT_STRUCT(stan::lang::double_literal,
                           (double,val_)
-                          (stan::lang::expr_type,type_) );
+                          (stan::lang::expr_type,type_) )
 
 
 namespace stan { 

